scripts/run_lighthouse_tests.py

```diff
@@ -107,7 +107,7 @@ def main():
     atexit.register(cleanup)
 
     python_utils.PRINT('Building files in production mode.')
-    build.main(args=['--prod_env'])
+    build.main(args=['--prod_env', '--source_maps'])
```

Comment:
I think we do not need `--source_maps` here.

Reply:
I did because i want atleast one ci check to have the `--source_maps` in prod_env to check if there are no issues when deploying.

Reply:
Hmmm, okay, but I would prefer if you used some e2e test and also this needs to be documented.

Reply:
Are you talking about this flag to be documented? Then please tell me which wiki page to document this. (I couldn't find any suitable page). Also I'll mail oppia-dev about it once it's merged.

Reply:
Also, I think we can keep this in the lighthouse checks itself given that there is already a special config for `terser` in e2e tests.

Reply:
Sorry, I meant to have a comment in the code explaining why we use the flag here since if it is not really needed for the tests I can imagine someone refactoring the tests and thinking it can be removed if there is no comment in the code.

Reply:
done